framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -262,7 +315,8 @@ public void run() {
   }
   
   public synchronized void setTestStatus(TestStatus status) {
-	if (testStatus == TestStatus.CANCELED) return;
+	if (testStatus == TestStatus.CANCELED || testStatus == TestStatus.VERIFICATION_FAILURE) 
```

Comment:
Yes, necessary. We may have other verification try to set it to success. It's a good defensive coding.